### PR TITLE
add try and catch for script

### PR DESCRIPTION
**Pull request by BlueShells**

### CODE REVIEW

File: scripts/monitoring/cron-send-logging-checks.py [hunk 1]
```diff
@@ -344,11 +344,18 @@ def run(self):
...
-        logging_status['kibana'] = self.check_kibana()
+        try:
+            logging_status['elasticsearch'] = self.check_elasticsearch()
+            logging_status['fluentd'] = self.check_fluentd()
+            logging_status['kibana'] = self.check_kibana()
+            self.report_to_zabbix(logging_status)
+        except:
+            result = 0 #something wrong with the checking script
+            mts = MetricSender(verbose=self.args.verbose)
+            mts.add_metric({'openshift.master.logging.elasticsearch.script.status': result})
```
Comment (wanghaoran1988) [inline scripts/monitoring/cron-send-logging-checks.py]: new metrics item? 0 stands for bad ? if there is no exception, should we send the metrics ? and what's the value?

Comment (BlueShells) [reply scripts/monitoring/cron-send-logging-checks.py]: its dynamic ones , too complex , its far more easy to create a new one

File: scripts/monitoring/cron-send-logging-checks.py [hunk 2]
```diff
@@ -344,11 +344,18 @@ def run(self):
...
-        logging_status['elasticsearch'] = self.check_elasticsearch()
-        logging_status['fluentd'] = self.check_fluentd()
-        logging_status['kibana'] = self.check_kibana()
+        try:
+            logging_status['elasticsearch'] = self.check_elasticsearch()
+            logging_status['fluentd'] = self.check_fluentd()
+            logging_status['kibana'] = self.check_kibana()
+            self.report_to_zabbix(logging_status)
+        except:
+            result = 0 #something wrong with the checking script
```
Comment (drewandersonnz) [inline scripts/monitoring/cron-send-logging-checks.py]: how does this get set to something which `!= 0` ?? If this script errors, there is no "cleared" state.

**Review by drewandersonnz — CHANGES_REQUESTED** [review]
-1, there is no clear state in the script. It will error on a fault, but will not remove the error when the fault clears.

**Review by drewandersonnz — APPROVED** [review]

**Pull request merged**